runtime/moonbase/tests/xcm_tests.rs

```diff
@@ -1833,118 +1833,11 @@ fn test_statemint_like() {
 		));
 	});
 
-	Statemint::execute_with(|| {
```

Comment:
This test was using the old prefix, to test we were able to receive it as well. I chose to remove it, but I can maintain it and asset the token is not received if we prefer